Intersect.Client/MonoGame/Graphics/MonoRenderer.cs

```diff
@@ -246,16 +244,16 @@ public override bool Begin()
                 UpdateGraphicsState(mScreenWidth, mScreenHeight);
             }
 
-            StartSpritebatch(mCurrentView, GameBlendModes.None, null, null, true, null);
+            StartSpritebatch(mCurrentView, GameBlendModes.None, null, null, true);
```

Comment:
Don't remove the `, null`

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/210284598-d591abf3-5f3a-47cc-b272-f01e496a6fe6.png)

Reason ^

Restoring this back anyways.